engine/install.go

```diff
@@ -123,6 +128,22 @@ poller:
 		Str("image", i.image).
 		Msg("create agent container")
 
+	keepaliveTimeEnv := os.Getenv("DRONE_AGENT_KEEPALIVE_TIME")
+	if keepaliveTimeEnv != "" {
```

Comment:
this data was already parsed when the `Config` structure was loaded by the system, and should not need to be re-parsed. Also setting `i.keepaliveTime` this way can cause a race condition, and could result in the program crashing.